src/components/MediaViewer/MediaViewer.js

```diff
@@ -0,0 +1,215 @@
+    }, [handleBackButtonPress] )
+  );
+
+  const deletePhoto = ( ) => {
+    deletePhotoFromObservation( photoUris[selectedPhotoIndex] );
+    hideWarningSheet( );
+    if ( photoUris.length === 0 ) {
+      navigation.goBack( );
+    } else if ( selectedPhotoIndex !== 0 ) {
+      setSelectedPhotoIndex( selectedPhotoIndex - 1 );
+    }
+  };
+
+  useEffect( ( ) => {
+    const renderHeaderTitle = ( ) => (
+      <Heading4 className="color-white">{t( "X-PHOTOS", { photoCount: numOfPhotos } )}</Heading4>
+    );
+
+    const headerOptions = {
+      headerTitle: renderHeaderTitle,
+      headerLeft: renderBackButton
+    };
+
+    navigation.setOptions( headerOptions );
+  }, [navigation, t, numOfPhotos, renderBackButton] );
+
+  const scrollToIndex = useCallback( index => {
+    // when a user taps a photo in the carousel, the UI needs to automatically
+    // scroll to the index of the photo they selected
+    setSelectedPhotoIndex( index );
+    horizontalScroll?.current?.scrollToIndex( { index, animated: true } );
+  }, [setSelectedPhotoIndex] );
+
+  const handleArrowPressLeft = ( ) => {
+    if ( atFirstPhoto ) { return; }
+    scrollToIndex( selectedPhotoIndex - 1 );
+  };
+
+  const handleArrowPressRight = ( ) => {
+    if ( atLastPhoto ) { return; }
+    scrollToIndex( selectedPhotoIndex + 1 );
+  };
+
+  const handleScrollEndDrag = e => {
+    const { contentOffset, layoutMeasurement } = e.nativeEvent;
+    const { x } = contentOffset;
+
+    const currentOffset = screenWidth * selectedPhotoIndex;
+
+    // https://gist.github.com/dozsolti/6d01d0f96d9abced3450a2e6149a2bc3?permalink_comment_id=4107663#gistcomment-4107663
+    const index = Math.floor(
+      Math.floor( x ) / Math.floor( layoutMeasurement.width )
+    );
+
+    if ( x > currentOffset ) {
+      handleScrollRight( index );
+    } else if ( x < currentOffset ) {
+      handleScrollLeft( index );
+    }
+  };
+
+  return (
+    <SafeAreaView className="flex-1 bg-black">
+      <StatusBar barStyle="light-content" />
+      {warningSheet && (
+        <WarningSheet
+          handleClose={hideWarningSheet}
+          confirm={deletePhoto}
+          headerText={t( "DISCARD-MEDIA" )}
+          snapPoints={[178]}
+          buttonText={t( "DISCARD" )}
+          secondButtonText={t( "CANCEL" )}
+          handleSecondButtonPress={hideWarningSheet}
+        />
+      )}
+      <MainPhotoDisplay
+        photoUris={photoUris}
+        selectedPhotoIndex={selectedPhotoIndex}
+        handleScrollEndDrag={handleScrollEndDrag}
+        horizontalScroll={horizontalScroll}
+      />
+      {!atFirstPhoto && (
+        <View className="absolute top-1/2 -mt-20 left-0">
+          <IconButton
+            onPress={handleArrowPressLeft}
+            icon="chevron-left-circle"
+            iconColor={colors.white}
+            size={26}
+          />
+        </View>
+      )}
+      {!atLastPhoto && (
+        <View className="absolute top-1/2 -mt-20 right-0">
```

Comment:
I think here and for the other button you want `-mt-[20px]` right? `-mt-20` is the equivalent of `-mt-[80px]` I think.